### DISCUSSION TITLE
Feature request - have a way to clear all checkmarks for a specific bunch

### DISCUSSION THREAD

**Discussion by tinylearnings** [ttscoff/bunch-docs#218] [Ideas]
Hi Brett,

First of all - I LOVE Bunch. Thank you so much for creating it.

Feature request: I have a "Close Everything" Bunch that just has the following:

---
title: ⚠️ Close EVERYTHING
menu order: 200
---

(quit everything)

@@

(show desktop)

After I run this bunch (usually end of day) - I always go in to "Clear Checkmarks". I was wondering if it would be possible to to have a character sequence that also clears all checkmarks (similar to - XX for browser). For example, using the above script - something like:

---
title: ⚠️ Close EVERYTHING
menu order: 200
---

(quit everything)
(clear checkmarks)

@@

(show desktop)

Thanks so much for everything!

**Comment by ttscoff** [comment]
I'll definitely consider this, seems like a worthwhile companion to 
"quit everything." I don't want to build it into the quit everything 
command as that command doesn't actually close any Bunches or perform 
any on-close actions, so it's not reasonable to assume they're closed, 
but I can see reasons why you'd want to "force" that.

-Brett

**Reply by tinylearnings** [reply]
Hi @ttscoff - thank you for considering this feature. I definitely don't think it should be built into "quit everything", but rather have it as a discrete option.
